joomla/joomla-cms
# Change routing strings to experimental and stable

### Summary of Changes

Change routing strings to experimental and stable on request by the release lead for 3.7.0 cc: @rdeutz 

### Testing Instructions

- Go to the settings where you can change the routing options
- confirm the new strings

### Expected result

Experimental + Stable

### Actual result

Modern + Legacy
